src/app/github-link-area/examples/github-link-area-example.component.ts

```diff
@@ -20,16 +20,30 @@ import {
 })
 export class GitHubLinkAreaExampleComponent implements OnInit {
 
-  content: string = `There is some text in here
-    <a href="https://github.com/patternfly/patternfly-ng/issues/127">https://github.com/patternfly/patternfly-ng/issues/127</a>. 
+  content: string = `There is some text in here.
+    And some checkboxes:<ul>
+    <li><input type="checkbox" data-checkbox-index="0"></input>An Item.</li>
+    <li><input type="checkbox" checked="" data-checkbox-index="1"></input>Checked Item.</li>
```

Comment:
@michaelkleinhenz what should checked="false" do ?
<input type="checkbox" checked="false" data-checkbox-index="1"></input>Checked Item.</li>

Reply:
Leftover. I'll remove that. :-)